app/grandchallenge/evaluation/templates/evaluation/evaluation_detail.html

```diff
@@ -286,23 +315,12 @@ <h3 class="card-title">Predictions</h3>
                     <a href="data:text/plain;charset=utf-8,{{ predictions|json_dumps|urlencode }}"
                         download="predictions.json"
                         class="btn btn-primary">
-                        <i class="fa fa-download"></i>
+                        <i class="fa fa-download"></i>&nbsp;&nbsp;
```

Comment:
Are the non-breakable spaces really necessary?

Reply:
Yes, do not tackle layout with spaces, breaks, tables, etc. Remove it so it is consistent with the rest of the site.

Reply:
Ok. I had actually added it to _make_ it consistent with the major UX item on the page: the challenge nav bar ( 😅 ). I'll smack that as well.